js/common.js

```diff
@@ -335,6 +335,15 @@ $(document).ready( function() {
 	$('a.click-url').bind("click", function() {
 		$(this).attr("href", $(this).attr("url"));
 	});
+
+	$('input[name=private].ace').bind("click", function() {
+		if ($(this).is(":checked")){
+			$('textarea[name=bugnote_text]').css("background-color", "#fcf8e3");
```

Comment:
i'd rather change a class in the textarea/container, than harcode color codes in javascript
Then in css, this class would define background colors